roles/network_plugin/calico/defaults/main.yml

```diff
@@ -143,7 +143,7 @@ calico_allow_ip_forwarding: false
 calico_ipam_strictaffinity: false
 
 # Calico IPAM autoAllocateBlocks
-calcio_ipam_autoallocateblocks: true
+calico_ipam_autoallocateblocks: true
```

Comment:
Since this was a typo and some poor soul might have relied on it, it would make sense to add a check and fail if the old typo is found so deployers can update their inventory variables.